_docs/developer/php_unit_tests.md

```diff
@@ -23,17 +25,34 @@ composer install
 This will download PHPUnit and PHPUnit-Selenium to the local machine
 and make it available via the vendor directory.
 
-Note: If you're using PHP 5.5, you'll first need to run the command
-
+If composer fails, you may be missing some required PHP modules.
+php-cli, php-mbstring, and php-xml should be installed already. 
+On Ubuntu/Debian these can be installed by:
+```
+sudo apt-get install php7.X-cli
+sudo apt-get install php7.X-mbstring
+sudo apt-get install php7.X-xml
 ```
-cp composer_55.json composer.json
+Replace `php-7.X` with the version of PHP installed locally. For example
+if you have PHP 7.2 then your command would like: `sudo apt-get install php7.2-cli`
+
+You will also need to have [Xdebug](https://xdebug.org/) locally installed.
+On Ubuntu/Debian you can install by:
 ```
+sudo apt-get install php7.X-xdebug
+``` 
 
-From there, to run the unit test suite, you can run from the root:
+From there, to run the unit test suite, you can run from `Submitty/site`:
 
 ```
 vendor/bin/phpunit --configuration tests/phpunit.xml
 ```
+If you see any errors running PHPUnit for the first time, you may be 
+missing some packages. 
+
+`curl_init()` errors can be solved by checking if `php-curl` is installed
+`ZipArchive` errors can be solved by checking if `php-zip` is installed
+`DataBase driver missing` can be solved by checking if `php-sqlite` is installed.
```

Comment:
You can add this one liner as an example to insall everything for a php7.2 env.
`apt-get -y install -q php7.2-cli php-curl php-zip php7.2-mbstring php7.2-xml php7.2-xdebug php7.2-sqlite3`